docs/userguides/cases.md

```diff
@@ -0,0 +1,48 @@
+# Cases
+
+Use py42 to quickly and easily manage cases.
+
+## Create, View or Modify Cases
+
+To get started create a new case.
+
+```python
+case = sdk.cases.create("new-case")
+```
+
+Once you've created a case, or if you're working with an existing one, you can use the case's `number` to view details about that case.
+
+```python
+
+# view details about a case
+case = sdk.cases.get(case_number)
+
+```
+
+You can also access a case by its `number` to update its details.  For instance, if you wanted to update a case's status to `CLOSED`:
+
+```python
+import py42.constants
+
+response = sdk.cases.update(case_number, status=py42.constants.CaseStatus.CLOSED)
```

Comment:
I think it's cleaner to import the constant class itself, especially if you need to use it in more than one place: 

```
from py42.constants import CaseStatus

response = sdk.cases.update(case_number, status=CaseStatus.CLOSED)
```